### DISCUSSION TITLE
Error generating a MIB

### DISCUSSION THREAD

**Discussion by alequadro** [prometheus/snmp_exporter#1195] [Q&A]
### Host operating system: output of `uname -a`
Linux devsecopslab01 5.15.0-107-generic #117-Ubuntu SMP Fri Apr 26 12:26:49 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux

### snmp_exporter version: output of `snmp_exporter -version`
docker image prom/snmp-exporter:v0.26.0

### What device/snmpwalk OID are you using?
N.A.

### If this is a new device, please link to the MIB(s).
https://jira.observium.org/secure/attachment/15732/16.2.4.5-patch1-6_MIB.txt

### What did you do that produced an error?
copied the MIB in the mibs folder, then ran 
make generate

### What did you expect to see?
a new snmp.yaml with the MIB imported

### What did you see instead?
I received an error 

ts=2024-06-03T10:50:12.994Z caller=main.go:135 level=error msg="Failing on reported parse error(s)" help="Use 'generator parse_errors' command to see errors, --no-fail-on-parse-errors to ignore"

**Comment by SuperQ** [comment]
Please read the error and report back.

**Reply by alequadro** [reply]
Hi SuperQ, thanks for the fast answer. 
Here attached the output of "make generator parse_errors"

[output.txt](https://github.com/user-attachments/files/15545134/output.txt)
